src/genie/libs/parser/asa/show_failover.py

```diff
@@ -0,0 +1,576 @@
+
+        # Interface management (192.168.253.14): Normal (Waiting)
+        # Interface outside (0.0.0.0/fe80::e8f:c5ff:fe5f:d01): Unknown (Waiting)
+        # Interface outside (0.0.0.0/fe80::ecd:e9ff:fe5c:5301): Normal (Not-Monitored)
+        p14 = re.compile(
+            r'^Interface\s+(?P<interface>\w+)\s+\((?P<ipv4>[0-9\.]+)'
+            '(\/(?P<ipv6>[A-Za-z0-9:]+))?\):\s+(?P<state>\w+)\s+'
+            '\((?P<monitored_state>[A-Za-z0-9\-_]+)\)$'
+        )
+
+        # slot 0: ASAv hw/sw rev (/9.14(1)) status (Up Sys)
+        # slot 1: IPS5515 hw/sw rev (N/A/7.1(4)E4) status (Up/Up)
+        # slot 0: ASA5515 hw/sw rev (1.0/9.2(2)4) status (Up Sys)
+        p15 = re.compile(
+            r'^slot\s+(?P<slot>\d+):\s+(?P<model>[A-Za-z0-9\-_]+) +hw\/sw +rev\s+'
+            '\((?P<version>\S+)\) +status\s+\((?P<status>[A-Za-z0-9_\-\s\/]+)\)$'
+        )
+
+        #         Recv Q:         0       0       0
+        p16_1 = re.compile(r'^Recv +Q:\s+(?P<curr>\d+)\s+(?P<max>\d+)\s+(?P<total>\d+)$')
+
+        # Xmit Q:         0       0       0
+        p16_2 = re.compile(r'^Xmit +Q:\s+(?P<curr>\d+)\s+(?P<max>\d+)\s+(?P<total>\d+)$')
+
+        # Stateful Failover Logical Update Statistics
+        # Link : foupdateif GigabitEthernet0/3 (administratively down)
+        p17_1 = re.compile(r'^Stateful +Failover +Logical +Update +Statistics$')
+        p17_2 = re.compile(
+            r'^Link\s+:\s+(?P<name>[A-Za-z0-9]+)\s+'
+            '(?P<interface>(Lo\S*|Fa\S*|Gi\S*|Ten\S*|\S*(SL|VL)\S*|Se\S*|VoIP\S*|Configured))'
+            '( +\((?P<status>[A-Za-z0-0_\-\s]+)\))?$'
+        )
+
+        parsed_dict = {}
+
+        # Indicates if have reached the This host section or Other host section
+        is_this_host = False
+
+        # Indicated if we have reached the Stateful Failover Logical Update Statistics section
+        is_stateful_failover_statistics_section = False
+
+        for line in out.splitlines():
+            line = line.strip()
+
+            # Failover On
+            # Failover Off
+            m = p1.match(line)
+            if m:
+                parsed_dict.update({
+                    'failover_enabled': True if m.groupdict()['status'] == 'On' else False
+                })
+                continue
+
+            # Failover unit Secondary
+            # Failover unit Primary
+            m = p2.match(line)
+            if m:
+                parsed_dict.update({'failover_unit': m.groupdict()['is_primary']})
+                continue
+
+            # Failover LAN Interface: folink GigabitEthernet0/1 (up)
+            m = p3.match(line)
+            if m:
+                group = m.groupdict()
+                if group['name'] == 'not':
+                    failover_lan_interface = {
+                        'status': 'not Configured'
+                    }
+                else:
+                    failover_lan_interface = {
+                        'name': group['name'],
```

Comment:
If you're interested, there's a tool that converts shorthand interface names to the full version 

https://github.com/CiscoTestAutomation/genieparser/blob/680ba11edc4859026cb0ce816160001a51b25771/src/genie/libs/parser/utils/common.py#L537